### DISCUSSION TITLE
Inquiry on Sionna's Support for Non-Terrestrial Network Simulation

### DISCUSSION THREAD

**Discussion by as057923553** [NVlabs/sionna#655] [General]
Hello,
I would like to ask if Sionna offers any tutorials for simulating Non-Terrestrial Networks. Currently, I have been learning about Sionna RT from [Introduction to Sionna RT](https://nvlabs.github.io/sionna/examples/Sionna_Ray_Tracing_Introduction.html). I hope to adapt the simulation from ground-based scenarios to Non-Terrestrial Networks, where a satellite in space transmits signals through the ionosphere and other environmental factors to reach a ground-based station for reception. This would involve the Doppler effect and environmental variations.

**Comment by SebastianCa** [comment]
Hi @as057923553,
there is no tutorial on NTN at the moment. However, you can apply the Doppler effect manually (see [tutorial example](https://nvlabs.github.io/sionna/examples/Sionna_Ray_Tracing_Introduction.html#From-Paths-to-Channel-Impulse-Responses)) or use SionnaRT [with mobility](https://nvlabs.github.io/sionna/examples/Sionna_Ray_Tracing_Mobility.html). please note that other environmental effects are currently not modelled and the ray tracing results would probably boil down to a LoS condition.

Moving this thread to discussions.

**Reply by as057923553** [reply]
Hi @SebastianCa 
Thanks for the information!
